src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -98,6 +98,41 @@ export class QuestionSubmissionFormComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.sortRecepientsByName();
+  }
+
+  /**
+   * Sorts recepients of feedback by their name.
+   *
+   * Feedback recepients are initially sorted by their email address.
+   */
+  private sortRecepientsByName(): void {
+    this.model.recipientList.sort((firstRecepient: FeedbackResponseRecipient,
+      secondRecepient: FeedbackResponseRecipient) => {
+      const firstRecepientNameLowerCase: string = firstRecepient.recipientName;
+      const secondRecepientNameLowerCase: string = secondRecepient.recipientName;
+
+      if (firstRecepientNameLowerCase < secondRecepientNameLowerCase) {
+        return -1;
+      }
+
+      if (firstRecepientNameLowerCase > secondRecepientNameLowerCase) {
+        return 1;
+      }
+
+      return 0;
+    });
+
+    this.model.recipientSubmissionForms.sort((firstRecepient: FeedbackResponseRecipientSubmissionFormModel,
+      secondRecepient: FeedbackResponseRecipientSubmissionFormModel) => {
+      const indexOne: number = this.model.recipientList.findIndex((x: FeedbackResponseRecipient) =>
+        x.recipientIdentifier === firstRecepient.recipientIdentifier);
+
+      const indexTwo: number = this.model.recipientList.findIndex((x: FeedbackResponseRecipient) =>
+        x.recipientIdentifier === secondRecepient.recipientIdentifier);
+
+      return indexOne - indexTwo;
```

Comment:
this block can also be replaced, same as above

Reply:
I think this is not applicable, as `FeedbackResponseRecipientSubmissionFormModel` goes by a `recipientIdentifier` which is not the name of the user but their email address, therefore we are using the sorted list from above to determine its index.

Reply:
I see, then I think we should construct a hashmap for `FeedbackResponseRecipientSubmissionFormModel` and name, because this current approach incurs extra O(n) with each comparison